en/docs/reference/synapse-properties/synapse-expressions.md

```diff
@@ -0,0 +1,777 @@
+# Synapse Expressions
+
+## Overview
+
+Synapse Expressions is a powerful expression designed to streamline the handling of JSON payloads. Handling of XML payloads is also supported using a dedicated function to evaluate XPATH expressions.
+
+Since, JSONPath is limited to extracting JSON content only, it's necessary to use XPath for performing additional operations on the extracted data (e.g., addition, substring extraction). Synapse Expressions addresses this limitation by enabling direct operations on JSON payloads, eliminating the need for complex workarounds.
```

Comment:
```suggestion
Since JSONPath is limited to extracting JSON content only, it's necessary to use XPath to perform additional operations on the extracted data (for example, addition and substring extraction). Synapse Expressions addresses this limitation by enabling direct operations on JSON payloads, eliminating the need for complex workarounds.
```